packages/server/package.json

```diff
@@ -20,7 +20,7 @@
         "pino-pretty": "^7.6.1"
     },
     "devDependencies": {
-        "@hashgraph/hedera-local": "^1.0.1",
+        "@hashgraph/hedera-local": "github:hashgraph/hedera-local-node",
```

Comment:
Q: Was this to just ensure latest bits for now?
It may not be necessary since the tests populate the desired docker version for the configurable local nodes

Reply:
It is necessary, at least until a new version of the package is published to npm. In v1.0.1 the env var `MIRROR_IMAGE_TAG` doesn't exist and the mirror node image version is hardcoded.

Reply:
You're right. Good catch